scripts/test

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+set -eufo pipefail
+
+# shellcheck source=/dev/null
+source "${BASH_SOURCE%/*}/../node_modules/sanctuary-scripts/functions"
+
+branches="$(get min-branch-coverage)"
+
+c8 --check-coverage \
+   --branches "$branches" \
+   node --experimental-modules \
+        --no-warnings \
+        ./node_modules/.bin/oletus
```

Comment:
@davidchambers maybe `sanctuary-scripts` could have `mocha` and `nyc` as _optional_ dependencies, so projects like this don't have to install a lot of deps for no reason.

Reply:
Good idea!